sql/database.sql

```diff
@@ -7716,7 +7716,7 @@ CREATE TABLE `prescriptions` (
   `quantity` varchar(31) default NULL,
   `size` varchar(25) DEFAULT NULL,
   `unit` int(11) default NULL,
-  `route` int(11) default NULL,
+  `route` varchar(25) default NULL,
```

Comment:
This change worries me! There's a reason this is an integer that only the original dev knows however we can't know what ramifications this change will have on other aspects of the module e.g. reporting, laminas module or validation.

Reply:
another way to approach it could be to grab the list_id instead in [the template](https://github.com/openemr/openemr/blob/master/templates/prescription/general_edit.html#L231)